instrumentation/xxl-job/xxl-job-1.9.2/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,35 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("com.xuxueli")
+    module.set("xxl-job-core")
+    versions.set("[1.9.2, 2.1.2)")
+    // except these versions, they are too old and the capabilities provided are not comprehensive.
+    skip("1.7.0", "1.7.1", "1.7.2", "1.8.0", "1.8.1", "1.8.2", "1.9.0", "1.9.1")
```

Comment:
> Usually we use `skip` for versions that are broken (for example they use dependencies that are not available in central). I guess your intent here is to disable all versions that are before `1.9.2` but where muzzle still passes. One way to get around this is to test for presence of some class that was added in a particular version for example see
> 
> https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/b5bbc62fa15c571ac6f22f40deb43b26b3690ec9/instrumentation/cassandra/cassandra-4.4/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/cassandra/v4_4/CassandraClientInstrumentationModule.java#L30-L33
> 
> Another alternative to artificially limit the supported versions would be to add a method like
> ```java
>   @SuppressWarnings({"Unused", "ReturnValueIgnored"})
>   private static void limitSupportedVersions() {
>     // GLUE_POWERSHELL was added in 1.9.2. Using this constant here ensures that muzzle will disable
>     // this instrumentation on earlier versions where this constant does not exist.
>     GlueTypeEnum.GLUE_POWERSHELL.name();
>   }
> ```
> 
> to `XxlJobSingletons` (I don't think we use this elsewhere)

The reason is as you guessed, I modified it according to your second suggestion, because, I didn't found suitable class to solve it by the first.